internal/worker/dbaccessor/worker.go

```diff
@@ -353,21 +328,20 @@ func (w *dbWorker) loop() (err error) {
 
 			req.done <- nil
```

Comment:
Do we know why this doesn't have a select guard (catacomb dying etc) around it?

Reply:
You wrote it :)